addons/copy-message-link/addon.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Copy link button",
+  "description": "Adds \"Copy Link\" button to comments.",
+  "userscripts": [{
+    "url": "userscript.js",
+    "matches": ["https://scratch.mit.edu/*"],
+    "runAtComplete": false
+  }],
+  "settings": [{
+      "name": "Add button besides Reply button",
+      "id": "reply",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Add button besides Report button",
+      "id": "report",
+      "type": "boolean",
+      "default": true
+    }
+  ],
```

Comment:
Make this one bool or make it a select type.

Reply:
Which one? They are already bools.

Reply:
Yeah... make it either one bool or make it a select type.